stoplight.js

```diff
@@ -12,13 +11,13 @@ $yellow.on('click', function() {
   $red.removeClass("active");
   $green.removeClass("active");
   $yellow.toggleClass("active");
-  setInterval(function () {
+  var myInterval = setTimeout(function () {
     $red.addClass("active");
     $yellow.removeClass("active");
   }, 2000);
```

Comment:
👍 good update. May want to keep `myInterval` in a higher scope so that it's available outside of this function.

Also could consider a function that removes the active class from each light and call that at the beginning of each listener